app/editor/src/features/content/form/components/upload/Upload.tsx

```diff
@@ -140,14 +140,20 @@ export const Upload: React.FC<IUploadProps> = ({
       <Show visible={!!stream && contentType === ContentTypeName.AudioVideo && !!fileReference}>
         <Show visible={fileReference?.contentType.startsWith('audio/')}>
           <Col flex="1">
-            <audio src={stream?.url} controls>
+            <audio controls>
+              {stream ? (
+                <source src={`${stream?.url}`} type={`${fileReference?.contentType}`} />
```

Comment:
Safari apparently needs the `type` parameter to work.  Once deployed we'll know for certain.